docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -91,6 +91,14 @@ Example pod templates
 
 With these requirements in mind, here are some examples of basic ``pod_template_file`` YAML files.
 
+.. note::
+
```

Comment:
I'm not sure these examples work as-is anywhere - they should be used for inspiration.

I'd be in favor of stating these are for inspiration and that care should be taken to ensure all configuration is consistent across components in your deployment.

Reply:
> I'd be in favor of stating these are for inspiration and that care should be taken to ensure all configuration is consistent across components in your deployment.

This is precisely the first thing you see when you follow the "config" link: https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html

| Use the same configuration across all the Airflow components. While each component does not require all, some configurations need to be same otherwise they would not work as expected. A good example for that is secret_key which should be same on the Webserver and Worker to allow Webserver to fetch logs from Worker.

Reply:
Maybe we should use a different word than "configuration". I didn't mean specifically "Airflow configuration", I meant in general. Another example would be `imagePullSecrets` - if you need it for the scheduler etc, you'll need it here too.

I'm _more_ in favor of stating these are for inspiration instead of implying they work if you use the "defaults".

Maybe something along these lines:

The following `pod_template_file` examples are for inspiration. This template will be used to create your workers, so take care to ensure all of your Airflow configuration and Kubernetes secrets/mounts/etc are added so the worker pod can function properly.

Reply:
Hey @jedcunningham - how strong are you with those :) - Maybe you can propose alternative PR if you are not happy with that :)?

Reply:
I can always open another PR to refactor this. I think this area needs more attention regardless.